authors/apps/articles/tests/test_pagination.py

```diff
@@ -0,0 +1,22 @@
+from .base_test import TestBaseCase
+
+
+class PaginationTestCase(TestBaseCase):
+    def create_multiple_articles(self):
+        for i in range(21):
+            self.create_article()
+
+    def is_present(self, value):
+        response = self.client.get(self.create_list_article_url)
+        self.assertIn(value, response.data)
+
+    def test_article_pagination(self):
+        self.is_present('count')
```

Comment:
Hi move the `response = self.client.get(self.create_list_article_url)` in the test then the `is_present` pick both data and value. This will reduce the number of calls you make to get the list of articles

Reply:
Done